public/static/docs/command-reference/checkout.md

```diff
@@ -16,60 +16,49 @@ positional arguments:
 
 ## Description
 
-[DVC-files](/doc/user-guide/dvc-file-format) in a <abbr>project</abbr> specify
-which instance of each data file or directory should be used, with the checksums
-saved in the `outs` field. The `dvc checkout` command updates the workspace data
-to match with the <abbr>cached</abbr> files that correspond to those checksums.
-
-Using an SCM like Git, the DVC-files are kept under version control. At a given
-branch or tag of the SCM repository, the DVC-files will contain checksums for
-the corresponding data files kept in the cache. After an SCM command like
-`git checkout` is run, the DVC-files will change to the state at the specified
-branch or commit or tag. Afterwards, the `dvc checkout` command is required in
-order to synchronize the data files with the currently checked out DVC-files.
-
-This command must be executed after `git checkout` since Git doesn't track files
-that are under DVC control. For convenience a Git hook is available, simply by
-running `dvc install`, that will automate running `dvc checkout` after
-`git checkout`. See `dvc install` for more information.
-
-The execution of `dvc checkout` does:
-
-- Scan the `outs` entries in DVC-files to compare with the currently checked out
-  data files. The scanned DVC-files is limited by the listed `targets` (if any)
-  on the command line. And if the `--with-deps` option is specified, it scans
-  backward from the given `targets` in the corresponding
-  [pipeline](/doc/command-reference/pipeline).
-
-- For any data files where the checksum doesn't match their DVC-file entry, the
-  data file is restored from the cache. The link strategy used (`reflink`,
-  `hardlink`, `symlink`, or `copy`) depends on the OS and the configured value
-  for `cache.type` – See `dvc config cache`.
-
-Note that this command by default tries NOT to copy files between the cache and
-the workspace, using reflinks instead when supported by the file system. (Refer
-to
+[DVC-files](/doc/user-guide/dvc-file-format) are essentially placeholders that
+point to the actual data files or directories under DVC control. This command
+synchronizes the workspace data with the versions specified in the current
+DVC-files. DVC knows which data (<abbr>outputs</abbr>) to use because their
```

Comment:
again, don't see any reason to introduce the term outputs whatsoever here. The only reasonable place is below - when we talk about implementation details. `To be precise, DVC knows which file to put into workspace by reading the checksums saved in the `outs` field in the DVC-files`. Not sure that we need this sentence at all here. Move it to the implementation details?

Reply:
This is the only instance left of the term "output" (other than a pre-existing one by the end of the description which has a different context). I thought we agreed it was OK to at least mention it once so the terminology is used. See https://github.com/iterative/dvc.org/pull/864#discussion_r362992294

I don't see why it's only reasonable to use this term in deeper implementation details. It's just a quick parenthesis to establish the term. But OK, I moved the sentence below, into the 2nd bullet of the implementation details. You're right that it makes more sense to have that sentence there.

> `To be precise, DVC knows which file to put into workspace by reading the checksums saved in the outs field in the DVC-files.`

Is this version of the sentence your suggestion? Or is it coming from an older version? The current version is "DVC knows which data (<abbr>outputs</abbr>) to use because their checksums are saved in the `outs` fields inside the DVC-files."

Reply:
OK I assume this one is resolved, lmk otherwise. Thanks 🙂